src/sql/parts/notebook/models/notebookModel.ts

```diff
@@ -253,10 +254,11 @@ export class NotebookModel extends Disposable implements INotebookModel {
 		try {
 			this._trustedMode = isTrusted;
 			let contents = null;
-			if (this._notebookOptions.notebookUri.scheme !== Schemas.untitled) {
-				// TODO: separate ContentManager from NotebookManager
-				contents = await this.notebookManagers[0].contentManager.getNotebookContents(this._notebookOptions.notebookUri);
-			}
+
+			let notebookEditorModel = await this._notebookParams.input.resolve();
```

Comment:
Can you add TODO on this to track need to have a better story than doing a `new LocalContentManager` each time? And add issues to track future removal of the contentManager APIs from the sqlops / azdata modules? We need to stop doing stuff with these in the MainThreadNotebook, SqlNotebookProvider etc.

That's OK to go in future PR, but let's track the work and plan ahead.

Reply:
https://github.com/Microsoft/azuredatastudio/issues/4323